modules/runtime-info/src/main/java/org/opencastproject/runtimeinfo/RuntimeInfo.java

```diff
@@ -218,6 +233,149 @@ public String getMyInfo() {
     return gson.toJson(result);
   }
 
+  @GET
+  @Path("health")
+  @Produces("application/health+json")
+  @RestQuery(name = "health", description = "Opencast node health check. Implements this internet-draft health check api https://inadarei.github.io/rfc-healthcheck",
+          reponses = {
+            @RestResponse(responseCode = HttpServletResponse.SC_OK, description = "Node is running, check reponse for details"),
+            @RestResponse(responseCode = HttpServletResponse.SC_SERVICE_UNAVAILABLE, description = "Node is offline or unresponsive, check response for details")},
+          returnDescription = "Details of the Opencast node's health status")
+
+  public String getHealth(@Context HttpServletResponse response) {
+    /* Response implements https://inadarei.github.io/rfc-healthcheck
```

Comment:
This, IMO, should be in the markdown docs somewhere, and linked to.

Reply:
surprised I didn't add it myself!